<!doctype html><html lang="en"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/milligram/1.4.1/milligram.css"><head><meta charset="utf-8"><meta name="Description" content="Research group of Andrew White at University of Rochester in the Department of Chemical Engineering"><title>wl</title><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=yes"><link rel="stylesheet" href="index.ee8d2c0e.css"></head><body> <div class="container"> <div class="row top"> <h1>whitelab@rochester</h1> </div> <p class="bigger"> <a href="#papers">papers</a> | <a href="https://scholar.google.com/citations?hl=en&user=47fQG30AAAAJ&view_op=list_works&sortby=pubdate">scholar</a> | <a href="https://github.com/ur-whitelab">code</a> | <a aria-label="Chemical Engineering at University of Rochester" href="https://www.hajim.rochester.edu/che/">che@ur</a> | <a href="https://whitead.github.io/dmol-book/">book</a> | <a href="https://whitead.github.io/numerical_stats/">class</a> <br> </p> <table> <tbody> <tr> <td>research</td> <td>large language models, chemistry deep learning, molecular dynamics</td> </tr> <tr>note: andrew is currently on sabbatical</tr> <tr> <td>phd</td> <td> <span>jorge&nbsp;medina&nbsp;&lt;jmedina9@ur.rochester.edu&gt;,</span> <span>sam&nbsp;cox&nbsp;&lt;swrig30@ur.rochester.edu&gt;,</span> <span>shane&nbsp;smichtavy&nbsp;&lt;smichtav@che.rochester.edu&gt;,</span> <span>quintina&nbsp;campbell&nbsp;&lt;qcampbe2@ur.rochester.edu&gt;</span> </td> </tr> <tr> <td>pi</td> <td>andrew white &lt;andrew.white@rochester.edu&gt;</td> </tr> <tr> <td>bio</td> <td>Andrew White is a researcher with over 50 peer-reviewed publications and books across the domains of large language models in chemistry, explainable artificial intelligence, statistical mechanics, and chemical engineering. He has won junior investigator awards from the National Science Foundation and National Institutes of Health along with professional and teaching awards for excellence as a chemical engineer. Andrew is an active member of the scientific community as a peer reviewer for over 30 journals, multiple national and private grant awarding institutions, and serves on the Chemical Sciences Roundtable at the National Academy of Science. Andrew is also a science communicator with large followings on X and LinkedIn and has been interviewed in multiple publications such as the New York Times, Bloomberg, Nature, Financial Times, and Science. Andrew serves on multiple scientific advisory boards across biotech. He has contributed to the ongoing debate around safety of artificial intelligence as a GPT-4 red teamer, speaking at multiple policy summits, and visiting the White House to advise multiple agencies. </td> </tr> <tr> <td>X</td> <td> <a href="https://x.com/andrewwhite01">@andrewwhite01</a>, <a href="https://x.com/SamCox822">@samcox822</a>, <a href="https://x.com/MichtavyShane">@michtavyshane</a>, <a href="https://x.com/quinnycampbell">@quinnycampbell</a>, <a href="https://x.com/4everstudent95">@4everstudent95</a> </td> </tr> <tr> <td>Bluesky</td> <td> <a href="https://bsky.app/profile/andrew.diffuse.one">@andrew.diffuse.one</a>, <a href="https://bsky.app/profile/quinnycampbell.bsky.social"> @quinnycampbell.bsky.social</a> </td> </tr> <tr> <td>media coverage</td> <td>interviewed/discussed in <a href="https://www.technologyreview.com/2023/07/05/1075865/eric-schmidt-ai-will-transform-science/"> MIT Tech Review</a>, <a href="https://www.nature.com/articles/d41586-023-00816-5">Nature</a>, <a href="https://www.newscientist.com/article/2370923-chemists-are-teaching-gpt-4-to-do-chemistry-and-control-lab-robots/">New Scientist</a>, <a href="https://www.ft.com/content/0876687a-f8b7-4b39-b513-5fee942831e8">Financial Times</a>, <a href="https://www.nature.com/articles/d41586-023-00674-1">Nature Careers</a> </td> </tr> <tr> <td>acknowledgements</td> <td>LLNL Project.<br> previous: doe bes de-sc0023354, nsf cbet #1751471, nsf dge #1922591, nsf dmr #2103553, nih #R35GM137966, nsf che #1764415, nsf iis #2029095 </td> </tr> <tr> <td>alumni</td> <td> <span>mayk&nbsp;caldas&nbsp;&lt;<a href="https://x.com/maykcaldas">@maykcaldas</a>&gt;,</span> <span>ziyue&nbsp;yang&nbsp;&lt;<a href="https://x.com/ZiyueYang37">@ZiyueYang37</a>&gt;,</span> <span>geemi&nbsp;wellawate&nbsp;&lt;<a href="https://x.com/GWellawatte">@GWellawatte</a>&gt;,</span> <span>mehrad&nbsp;ansari&nbsp;&lt;<a href="https://x.com/MehradAnsari">@MehradAnsari</a>&gt;,</span> <span>heta&nbsp;gandhi&nbsp;&lt;<a href="https://x.com/gandhi_heta">@gandhi_heta</a>&gt;,</span> <span>rainier&nbsp;barrett&nbsp;&lt;<a href="https://x.com/Rainier_B">@Rainier_B</a>&gt;</span> </td> </tr> </tbody> </table> <p></p> <img id="pic" class="group" src="group.c1944b37.jpg" alt="Picture of the group members looking handsome and smart"> <p><a href="https://mol.dog" aria-label="mol dog">🧪.🐕</a> | <a href="https://whitead.github.io/molecule-dream/" aria-label="mol dream">⚗️💤</a> | <a href="https://whitead.github.io/graph-mcmc/" aria-label="Monte Carlo graph">⚡</a> | <a href="https://colab.research.google.com/github/whitead/emoji-chem/blob/main/colab/EmojiChem.ipynb" aria-label="emoji chem"> 🥳⚗️ </a>| <a href="https://colab.research.google.com/github/whitead/emoji-math/blob/master/colab/EmojiMath.ipynb" aria-label="emoji math"> 🥳🧮 </a> | <a href="https://colab.research.google.com/github/whitead/clipmol/blob/master/CLIPMol.ipynb" aria-label="clip mol"> 📎⚗️ </a> | <a href="hhttps://github.com/whitead/ternviz" aria-label="ternviz"> 🐦🎥 </a> | <a href="https://peptide.bio" aria-label="peptide dashboard">🌶️🌊</a> | <a href="https://github.com/whitead/molcloud" aria-label="molcloud">⚗️☁️</a> | <a href="https://mol.dev" aria-label="solubility predictor">🧪🌊</a> | <a href="https://github.com/whitead/synspace" aria-label="chemical space enumerator">0xe4cc</a> </p> <hr>  <h2 id="papers">papers</h2> <ol> <li> Ansari, M. & White, A. D. Learning peptide properties with positive examples only. Digital Discovery 3, 977–986 (2024).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1666104177360084995">tweet</a> | <a href="https://pubs.rsc.org/en/content/articlelanding/2024/dd/d3dd00218g">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2024/dd/d3dd00218g">pdf</a> | <a href="https://github.com/ur-whitelab/pu-peptides">code</a> </div> </li><li> Ramos, M. C. & White, A. D. Predicting small molecules solubility on endpoint devices using deep ensemble neural networks. Digital Discovery 3, 786–795 (2024).<div class="paper-info"><a href="https://x.com/andrewwhite01/status/1780944395656744962">tweet</a> | <a href="https://pubs.rsc.org/en/content/articlelanding/2024/dd/d3dd00217a">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2024/dd/d3dd00217a">pdf</a> | <a href="https://mol.dev/">app</a> | <a href="https://github.com/ur-whitelab/mol.dev">code</a> </div> </li><li> Cox, S., Hammerling, M., Lála, J., Laurent, J., Rodriques, S., Rubashkin, M. & White, A. WikiCrow: Automating synthesis of human scientific knowledge (2024).<div class="paper-info"><a href="https://x.com/SGRodriques/status/1733193844110557432">tweet</a> | <a href="https://www.futurehouse.org/wikicrow">link</a> | <a href="https://github.com/Future-House/WikiCrow">code</a> </div> </li><li> Ramos, M. C., Collison, C. J. & White, A. D. A review of large language models and autonomous agents in chemistry. arXiv preprint arXiv:2407.01603 (2024).<div class="paper-info"><a href="https://x.com/andrewwhite01/status/1808505284291375380">tweet</a> | <a href="https://pubs.rsc.org/en/content/articlelanding/2025/sc/d4sc03921a">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2025/sc/d4sc03921a">pdf</a> | <a href="https://github.com/ur-whitelab/LLMs-in-science">code</a> </div> </li><li> Tribello, G. A., Bonomi, M., Bussi, G., Camilloni, C., Armstrong, B. I., Arsiccio, A., Aureli, S., Ballabio, F., Bernetti, M., Bonati, L., Brookes, S. G. H., Brotzakis, Z. F., Capelli, R., Ceriotti, M., Chan, K.-T., Cossio, P., Dasetty, S., Donadio, D., Ensing, B., Ferguson, A. L., Fraux, G., Gale, J. D., Gervasio, F. L., Giorgino, T., Herringer, N. S. M., Hocky, G. M., Hoff, S. E., Invernizzi, M., Languin-Cattöen, O., Leone, V., Limongelli, V., Lopez-Acevedo, O., Marinelli, F., Martinez, P. F., Masetti, M., Mehdi, S., Michaelides, A., Murtada, M. H., Parrinello, M., Piaggi, P. M., Pietropaolo, A., Pietrucci, F., Pipolo, S., Pritchard, C., Raiteri, P., Raniolo, S., Rapetti, D., Rizzi, V., Rydzewski, J., Salvalaglio, M., Schran, C., Seal, A., Zadeh, A. S., Silva, T. F. D., Spiwok, V., Stirnemann, G., Sucerquia, D., Tiwary, P., Valsson, O., Vendruscolo, M., Voth, G. A., White, A. D. & Wu, J. PLUMED tutorials: A collaborative, community-driven learning ecosystem. (2024).<div class="paper-info"><a href="null">tweet</a> | <a href="https://arxiv.org/abs/2412.03595">link</a> | <a href="https://arxiv.org/pdf/2412.03595">pdf</a> </div> </li><li> Skarlinski, M. D., Cox, S., Laurent, J. M., Braza, J. D., Hinks, M., Hammerling, M. J., Ponnapati, M., Rodriques, S. G. & White, A. D. Language agents achieve superhuman synthesis of scientific knowledge. (2024).<div class="paper-info"><a href="https://x.com/andrewwhite01/status/1880372434551111825">tweet</a> | <a href="https://arxiv.org/abs/2409.13740">link</a> | <a href="https://arxiv.org/pdf/2409.13740">pdf</a> | <a href="https://github.com/future-house/paper-qa">code</a> </div> </li><li> Narayanan, S., Braza, J. D., Griffiths, R.-R., Ponnapati, M., Bou, A., Laurent, J., Kabeli, O., Wellawatte, G., Cox, S., Rodriques, S. G. & White, A. D. Aviary: Training language agents on challenging scientific tasks. (2024).<div class="paper-info"><a href="https://x.com/andrewwhite01/status/1874180131402768759">tweet</a> | <a href="https://arxiv.org/abs/2412.21154">link</a> | <a href="https://arxiv.org/pdf/2412.21154">pdf</a> | <a href="https://github.com/future-house/aviary">code</a> </div> </li><li> Tribello, G. A., Bonomi, M., Bussi, G., Camilloni, C., Armstrong, B. I., Arsiccio, A., Aureli, S., Ballabio, F., Bernetti, M., Bonati, L., Brookes, S. G. H., Brotzakis, Z. F., Capelli, R., Ceriotti, M., Chan, K.-T., Cossio, P., Dasetty, S., Donadio, D., Ensing, B., Ferguson, A. L., Fraux, G., Gale, J. D., Gervasio, F. L., Giorgino, T., Herringer, N. S. M., Hocky, G. M., Hoff, S. E., Invernizzi, M., Languin-Cattöen, O., Leone, V., Limongelli, V., Lopez-Acevedo, O., Marinelli, F., Martinez, P. F., Masetti, M., Mehdi, S., Michaelides, A., Murtada, M. H., Parrinello, M., Piaggi, P. M., Pietropaolo, A., Pietrucci, F., Pipolo, S., Pritchard, C., Raiteri, P., Raniolo, S., Rapetti, D., Rizzi, V., Rydzewski, J., Salvalaglio, M., Schran, C., Seal, A., Zadeh, A. S., Silva, T. F. D., Spiwok, V., Stirnemann, G., Sucerquia, D., Tiwary, P., Valsson, O., Vendruscolo, M., Voth, G. A., White, A. D. & Wu, J. PLUMED tutorials: A collaborative, community-driven learning ecosystem. (2024).<div class="paper-info"><a href="null">tweet</a> | <a href="https://arxiv.org/abs/2412.03595">link</a> | <a href="https://arxiv.org/pdf/2412.03595">pdf</a> </div> </li><li> Le, Z., Ramos, M. C., Shou, Y., Li, R. R., Cheng, H. S., Jang, C. J., Liu, L., Xue, C., Li, X., Liu, H., et al. Bioactive sucralfate-based microneedles promote wound healing through reprogramming macrophages and protecting endogenous growth factors. Biomaterials 311, 122700 (2024). </li><li> Ansari, M. & White, A. D. Serverless prediction of peptide properties with recurrent neural networks. Journal of Chemical Information and Modeling 63, 2546–2553 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1527667461608316928">tweet</a> | <a href="https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01317">link</a> | <a href="https://pubs.acs.org/doi/pdf/10.1021/acs.jcim.2c01317">pdf</a> | <a href="https://peptide.bio">app</a> | <a href="https://github.com/ur-whitelab/peptide-dashboard">code</a> </div> </li><li> Wellawatte, G. P., Gandhi, H. A., Seshadri, A. & White, A. D. A perspective on explanations of molecular prediction models. Journal of Chemical Theory and Computation 19, 2149–2160 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1601291757219897344">tweet</a> | <a href="https://pubs.acs.org/doi/10.1021/acs.jctc.2c01235">link</a> | <a href="https://pubs.acs.org/doi/epdf/10.1021/acs.jctc.2c01235">pdf</a> | <a href="https://github.com/ur-whitelab/exmol/">code</a> </div> </li><li> White, A. D., Hocky, G. M., Gandhi, H. A., Ansari, M., Cox, S., Wellawatte, G. P., Sasmal, S., Yang, Z., Liu, K., Singh, Y., et al. Assessment of chemistry knowledge in large language models that generate code. Digital Discovery 2, 368–376 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1602321379142995973">tweet</a> | <a href="https://pubs.rsc.org/en/content/articlehtml/2023/dd/d2dd00087c">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2023/dd/d2dd00087c">pdf</a> | <a href="https://github.com/ur-whitelab/nlcc-data">code</a> </div> </li><li> Wellawatte, G. P., Hocky, G. M. & White, A. D. Neural potentials of proteins extrapolate beyond training data. The Journal of Chemical Physics 159, (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1605937402509492225">tweet</a> | <a href="https://pubs.aip.org/aip/jcp/article/159/8/085103/2908350">link</a> | <a href="https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/63a3bfac81e4ba77ec56166a/original/neural-potentials-of-proteins-extrapolate-beyond-training-data.pdf">pdf</a> </div> </li><li> Bran, A. M., Cox, S., Schilter, O., Baldassari, C., White, A. D. & Schwaller, P. ChemCrow: Augmenting large-language models with chemistry tools. arXiv preprint arXiv:2304.05376 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1670794000398184451">tweet</a> | <a href="https://www.nature.com/articles/s42256-024-00832-8">link</a> | <a href="https://www.nature.com/articles/s42256-024-00832-8.pdf">pdf</a> | <a href="https://github.com/ur-whitelab/chemcrow-public/">code</a> | <a href="https://github.com/ur-whitelab/chemcrow-runs">outputs</a> </div> </li><li> Ramos, M. C., Michtavy, S. S., Porosoff, M. D. & White, A. D. Bayesian optimization of catalysts with in-context learning. arXiv preprint arXiv:2304.05341 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1645953720247615491">tweet</a> | <a href="https://arxiv.org/abs/2304.05341">link</a> | <a href="https://arxiv.org/pdf/2304.05341.pdf">pdf</a> | <a href="https://github.com/ur-whitelab/BO-LIFT">code</a> </div> </li><li> Medina, J. & White, A. D. Bloom filters for molecules. Journal of Cheminformatics 15, 95 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1646171800940969984">tweet</a> | <a href="https://link.springer.com/article/10.1186/s13321-023-00765-1%20">link</a> | <a href="https://link.springer.com/content/pdf/10.1186/s13321-023-00765-1.pdf">pdf</a> | <a href="https://github.com/whitead/molbloom">code</a> </div> </li><li> Campbell, Q. L., Herington, J. & White, A. D. Censoring chemical data to mitigate dual use risk. arXiv preprint arXiv:2304.10510 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1649237566594351104?s=20">tweet</a> | <a href="https://arxiv.org/abs/2304.10510">link</a> | <a href="https://arxiv.org/pdf/2304.10510.pdf">pdf</a> | <a href="https://github.com/ur-whitelab/chem-dual-use">code</a> </div> </li><li> Medina, J. & White, A. D. Active learning in symbolic regression performance with physical constraints. arXiv preprint arXiv:2305.10379 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1661450084335099904">tweet</a> | <a href="https://arxiv.org/abs/2305.10379">link</a> | <a href="https://arxiv.org/pdf/2305.10379">pdf</a> | <a href="https://github.com/Jgmedina95/SRwPhysConsWL">code</a> </div> </li><li> White, A. D. The future of chemistry is language. Nature Reviews Chemistry 7, 457–458 (2023).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1659547294377680896">tweet</a> | <a href="https://www.nature.com/articles/s41570-023-00502-0">link</a> | <a href="https://www.nature.com/articles/s41570-023-00502-0.pdf">pdf</a> </div> </li><li> Jablonka, K. M., Ai, Q., Al-Feghali, A., Badhwar, S., Bocarsly, J. D., Bran, A. M., Bringuier, S., Brinson, L. C., Choudhary, K., Circi, D., et al. 14 examples of how LLMs can transform materials science and chemistry: A reflection on a large language model hackathon. Digital Discovery 2, 1233–1250 (2023).<div class="paper-info"><a href="https://x.com/kmjablonka/status/1668506349465116673">tweet</a> | <a href="https://pubs.rsc.org/en/content/articlelanding/2023/dd/d3dd00113j">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2023/dd/d3dd00113j">pdf</a> </div> </li><li> Lo, A., Pollice, R., Nigam, A., White, A. D., Krenn, M. & Aspuru-Guzik, A. Recent advances in the self-referencing embedded strings (SELFIES) library. Digital Discovery 2, 897–908 (2023).<div class="paper-info"><a href="https://arxiv.org/abs/2302.03620">link</a> | <a href="https://arxiv.org/pdf/2302.03620.pdf">pdf</a> | <a href="https://github.com/aspuru-guzik-group/selfies">code</a> </div> </li><li> Lála, J., O’Donoghue, O., Shtedritski, A., Cox, S., Rodriques, S. G. & White, A. D. Paperqa: Retrieval-augmented generative agent for scientific research. arXiv preprint arXiv:2312.07559 (2023).<div class="paper-info"><a href="null">tweet</a> | <a href="https://doi.org/10.48550/arXiv.2312.07559">link</a> | <a href="https://arxiv.org/pdf/2312.07559">pdf</a> | <a href="https://github.com/Future-House/paper-qa">code</a> </div> </li><li> Barrett, R., Ansari, M., Ghoshal, G. & White, A. D. Simulation-based inference with approximately correct parameters via maximum entropy. Machine Learning: Science and Technology 3, 025006 (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1430357286556405761">tweet</a> | <a href="https://iopscience.iop.org/article/10.1088/2632-2153/ac6286/meta">link</a> | <a href="https://iopscience.iop.org/article/10.1088/2632-2153/ac6286/pdf">pdf</a> | <a href="https://github.com/ur-whitelab/maxent/">code</a> </div> </li><li> Wellawatte, G. P., Seshadri, A. & White, A. D. Model agnostic generation of counterfactual explanations for molecules. Chemical science 13, 3697–3705 (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1427268134654914560">tweet</a> | <a href="https://doi.org/10.1039/D1SC05259D">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2022/sc/d1sc05259d">pdf</a> | <a href="https://github.com/ur-whitelab/exmol">code</a> </div> </li><li> Ansari, M., Gandhi, H. A., Foster, D. G. & White, A. D. Iterative symbolic regression for learning transport equations. AIChE Journal 68, e17695 (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1424913806447292421">tweet</a> | <a href="https://doi.org/10.1002/aic.17695">link</a> | <a href="https://arxiv.org/pdf/2108.03293">pdf</a> | <a href="https://github.com/ur-whitelab/alcfd/">code</a> </div> </li><li> Hocky, G. M. & White, A. D. Natural language processing models that automate programming will transform chemistry research and teaching. Digital discovery 1, 79–83 (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1432720845638672390">tweet</a> | <a href="https://doi.org/10.1039/D1DD00009H">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2022/dd/d1dd00009h">pdf</a> | <a href="https://github.com/whitead/nlcc/">code</a> </div> </li><li> Ansari, M., Soriano-Paños, D., Ghoshal, G. & White, A. D. Inferring spatial source of disease outbreaks using maximum entropy. Physical Review E 106, 014306 (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1447361299407675396">tweet</a> | <a href="https://journals.aps.org/pre/abstract/10.1103/PhysRevE.106.014306">link</a> | <a href="https://arxiv.org/pdf/2110.03846.pdf">pdf</a> | <a href="https://github.com/ur-whitelab/py0">code</a> </div> </li><li> Hamsici, S., White, A. D. & Acar, H. Peptide framework for screening the effects of amino acids on assembly. Science Advances 8, eabj0305 (2022).<div class="paper-info"><a href="https://twitter.com/acar_handan/status/1484324063916953602">tweet</a> | <a href="https://www.science.org/doi/10.1126/sciadv.abj0305">link</a> | <a href="https://www.science.org/doi/pdf/10.1126/sciadv.abj0305?download=true">pdf</a> | <a href="https://www.plumed-nest.org/eggs/22/006/">code</a> </div> </li><li> Krenn, M., Ai, Q., Barthel, S., Carson, N., Frei, A., Frey, N. C., Friederich, P., Gaudin, T., Gayle, A. A., Jablonka, K. M., et al. SELFIES and the future of molecular string representations. Patterns 3, (2022).<div class="paper-info"><a href="https://twitter.com/MarioKrenn6240/status/1510781069423001608">tweet</a> | <a href="https://www.sciencedirect.com/science/article/pii/S2666389922002069">link</a> | <a href="https://www.sciencedirect.com/sdfe/reader/pii/S2666389922002069/pdf">pdf</a> | <a href="https://github.com/aspuru-guzik-group/selfies/">code</a> </div> </li><li> Cox, S. & White, A. D. Symmetric molecular dynamics. Journal of Chemical Theory and Computation 18, 4077–4081 (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1511138339998650378">tweet</a> | <a href="https://doi.org/10.1021/acs.jctc.2c00401">link</a> | <a href="https://arxiv.org/pdf/2204.01114">pdf</a> | <a href="https://github.com/whitead/symd">code</a> </div> </li><li> Kalinin, S. V., Ziatdinov, M., Sumpter, B. G. & White, A. D. Physics is the new data. arXiv preprint arXiv:2204.05095 (2022).<div class="paper-info"><a href="https://arxiv.org/abs/2204.05095">link</a> | <a href="https://arxiv.org/pdf/2204.05095">pdf</a> </div> </li><li> Zhu, W., Luo, J. & White, A. D. Federated learning of molecular properties with graph neural networks in a heterogeneous setting. Patterns 100521 (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1438470336182358017">tweet</a> | <a href="https://doi.org/10.1016/j.patter.2022.100521">link</a> | <a href="https://arxiv.org/pdf/2109.07258">pdf</a> | <a href="https://github.com/ur-whitelab/fedchem">code</a> </div> </li><li> White, A. D. Deep learning for molecules and materials. Living journal of computational molecular science 3, (2022). </li><li> Yang, Z., Milas, K. A. & White, A. D. Now what sequence? Pre-trained ensembles for bayesian optimization of protein sequences. bioRxiv 2022–08 (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1556358930288230400">tweet</a> | <a href="https://www.biorxiv.org/content/10.1101/2022.08.05.502972v1">link</a> | <a href="https://www.biorxiv.org/content/10.1101/2022.08.05.502972v1.full.pdf">pdf</a> | <a href="https://github.com/ur-whitelab/wazy">code</a> </div> </li><li> Gandhi, H. A. & White, A. D. Explaining molecular properties with natural language. (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1523604252006170630">tweet</a> | <a href="https://chemrxiv.org/engage/chemrxiv/article-details/633731d1f764e6e535093041">link</a> | <a href="https://github.com/ur-whitelab/exmol">code</a> </div> </li><li> Seshadri, A., Gandhi, H. A., Wellawatte, G. P. & White, A. D. Why does that molecule smell? (2022).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1600857844899024897">tweet</a> | <a href="https://chemrxiv.org/engage/chemrxiv/article-details/6390ebaab103af2aeb0ed11f">link</a> | <a href="https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6390ebaab103af2aeb0ed11f/original/why-does-that-molecule-smell.pdf">pdf</a> | <a href="https://github.com/ur-whitelab/exmol/tree/main/paper3_Scents">code</a> </div> </li><li> Yang, Z., Chakraborty, M. & White, A. D. Predicting chemical shifts with graph neural networks. Chemical Science (2021).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1414585891583053830">tweet</a> | <a href="https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc01895g">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2021/sc/d1sc01895g">pdf</a> | <a href="https://github.com/ur-whitelab/nmrgnn">code</a> </div> </li><li> Gandhi, H. A. & White, A. D. City-wide modeling of vehicle-to-grid economics to understand effects of battery performance. ACS Sustainable Chemistry & Engineering 9, 14975–14985 (2021).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1426178869263011842">tweet</a> | <a href="https://pubs.acs.org/doi/abs/10.1021/acssuschemeng.1c05490">link</a> | <a href="https://arxiv.org/pdf/2108.05837">pdf</a> | <a href="https://github.com/hgandhi2411/Vehicle_to_grid">code</a> </div> </li><li> Chakraborty, M., Xu, J. & White, A. D. Is preservation of symmetry necessary for coarse-graining? Physical Chemistry Chemical Physics 22, 14998–15005 (2020).<div class="paper-info"><a href="https://doi.org/10.1039/D0CP02309D">link</a> | <a href="https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/60c74618567dfe45edec457d/original/is-preservation-of-symmetry-necessary-for-coarse-graining.pdf">pdf</a> </div> </li><li> Chakraborty, M., Ziatdinov, M., Dyck, O., Jesse, S., White, A. D. & Kalinin, S. V. Reconstruction of the interatomic forces from dynamic scanning transmission electron microscopy data. Journal of Applied Physics 127, (2020).<div class="paper-info"><a href="https://doi.org/10.1063/5.0009413">link</a> </div> </li><li> Li, Z., Wellawatte, G. P., Chakraborty, M., Gandhi, H. A., Xu, C. & White, A. D. Graph neural network based coarse-grained mapping prediction. Chemical science 11, 9524–9531 (2020).<div class="paper-info"><a href="https://twitter.com/andrewwhite01/status/1422562950401990658">tweet</a> | <a href="https://pubs.rsc.org/en/content/articlehtml/2020/sc/d0sc02458a">link</a> | <a href="https://pubs.rsc.org/en/content/articlepdf/2020/sc/d0sc02458a">pdf</a> | <a href="https://github.com/rochesterxugroup/DSGPM">code</a> </div> </li><li> Tang, J., Zhang, Y., Luehmann, A. & White, A. Augmented reality improved learning of lower-level students by empowering their participation in collaborative activities. (2020).<div class="paper-info"><a href="https://repository.isls.org//handle/1/6846">link</a> | <a href="https://repository.isls.org/bitstream/1/6846/1/999-1006.pdf">pdf</a> </div> </li><li> Barrett, R., Chakraborty, M., Amirkulova, D., Gandhi, H., Wellawatte, G. & White, A. Hoomd-tf: Gpu-accelerated, online machine learning in the hoomd-blue molecular dynamics engine. Journal of Open Source Software 5, (2020).<div class="paper-info"><a href="https://doi.org/10.21105/joss.02367">link</a> | <a href="https://www.theoj.org/joss-papers/joss.02367/10.21105.joss.02367.pdf">pdf</a> | <a href="https://github.com/ur-whitelab/hoomd-tf/">code</a> </div> </li><li> Amirkulova, D. B., Chakraborty, M. & White, A. D. Experimentally consistent simulation of aβ21–30 peptides with a minimal NMR bias. The Journal of Physical Chemistry B 124, 8266–8277 (2020).<div class="paper-info"><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jpcb.0c07129">link</a> | <a href="https://pubs.acs.org/doi/pdf/10.1021/acs.jpcb.0c07129">pdf</a> </div> </li><li> Gandhi, H. A., Jakymiw, S., Barrett, R., Mahaseth, H. & White, A. D. Real-time interactive simulation and visualization of organic molecules. (2020).<div class="paper-info"><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jchemed.9b01161">link</a> | <a href="https://github.com/ur-whitelab/simview">code</a> </div> </li><li> Barrett, R. & White, A. D. Investigating active learning and meta-learning for iterative peptide design. Journal of chemical information and modeling 61, 95–105 (2020).<div class="paper-info"><a href="https://pubs.acs.org/doi/10.1021/acs.jcim.0c00946">link</a> | <a href="https://pubs.acs.org/doi/pdf/10.1021/acs.jcim.0c00946">pdf</a> | <a href="https://github.com/ur-whitelab/peptide-ai">code</a> </div> </li><li> Amirkulova, D. B. & White, A. D. Recent advances in maximum entropy biasing techniques for molecular dynamics. Molecular Simulation 45, 1285–1294 (2019).<div class="paper-info"><a href="https://www.tandfonline.com/doi/full/10.1080/08927022.2019.1608988">link</a> | <a href="https://www.tandfonline.com/doi/pdf/10.1080/08927022.2019.1608988">pdf</a> </div> </li><li> Promoting transparency and reproducibility in enhanced molecular simulations. Nature methods 16, 670–673 (2019).<div class="paper-info"><a href="https://twitter.com/plumed_org/status/1156225028096241665">tweet</a> | <a href="https://www.nature.com/articles/s41592-019-0506-8">link</a> | <a href="https://www.nature.com/articles/s41592-019-0506-8.pdf">pdf</a> | <a href="https://www.plumed-nest.org/">code</a> </div> </li><li> Mayes, H. B., Lee, S., White, A. D., Voth, G. A. & Swanson, J. M. Multiscale kinetic modeling reveals an ensemble of cl–/h+ exchange pathways in ClC-ec1 antiporter. Journal of the American Chemical Society 140, 1793–1804 (2018).<div class="paper-info"><a href="https://pubs.acs.org/doi/10.1021/jacs.7b11463">link</a> | <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5812667/pdf/ja7b11463.pdf">pdf</a> </div> </li><li> Amirkulova, D. B. & White, A. D. Combining enhanced sampling with experiment-directed simulation of the GYG peptide. Journal of Theoretical and Computational Chemistry 17, 1840007 (2018).<div class="paper-info"><a href="https://www.worldscientific.com/doi/abs/10.1142/S0219633618400072">link</a> | <a href="https://arxiv.org/pdf/1804.05100">pdf</a> </div> </li><li> Chakraborty, M., Xu, C. & White, A. D. Encoding and selecting coarse-grain mapping operators with hierarchical graphs. The Journal of Chemical Physics 149, (2018).<div class="paper-info"><a href="https://doi.org/10.1039/D0CP02309D">link</a> | <a href="https://chemrxiv.org/engage/chemrxiv/article-details/60c74618567dfe45edec457d">pdf</a> | <a href="https://github.com/ur-whitelab/mapping-graph">code</a> </div> </li><li> Barrett, R., Jiang, S. & White, A. D. Classifying antimicrobial and multifunctional peptides with bayesian network models. Peptide Science 110, e24079 (2018).<div class="paper-info"><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/pep2.24079">link</a> | <a href="https://arxiv.org/pdf/1804.06327">pdf</a> | <a href="https://github.com/ur-whitelab/peptide-qspr">code</a> </div> </li><li> Barrett, R., Gandhi, H. A., Naganathan, A., Daniels, D., Zhang, Y., Onwunaka, C., Luehmann, A. & White, A. D. Social and tactile mixed reality increases student engagement in undergraduate lab activities. Journal of Chemical Education 95, 1755–1762 (2018). </li><li> White, A. D., Knight, C., Hocky, G. M. & Voth, G. A. Communication: Improved ab initio molecular dynamics by minimally biasing with experimental data. The Journal of Chemical Physics 146, (2017). </li><li> Freeman, G. M., Drennen, T. E. & White, A. D. Can parked cars and carbon taxes create a profit? The economics of vehicle-to-grid energy storage for peak reduction. Energy Policy 106, 183–190 (2017). </li><li> Dannenhoffer-Lafage, T., White, A. D. & Voth, G. A. A direct method for incorporating experimental data into multiscale coarse-grained models. Journal of chemical theory and computation 12, 2144–2153 (2016). </li><li> White, A. D., Dama, J. F. & Voth, G. A. Designing free energy surfaces that match experimental data with metadynamics. Journal of Chemical Theory and Computation 11, 2451–2460 (2015). </li><li> Shao, Q., White, A. D. & Jiang, S. Difference of carboxybetaine and oligo (ethylene glycol) moieties in altering hydrophobic interactions: A molecular simulation study. The Journal of Physical Chemistry B 118, 189–194 (2014). </li><li> Nowinski, A. K., White, A. D., Keefe, A. J. & Jiang, S. Biologically inspired stealth peptide-capped gold nanoparticles. Langmuir 30, 1864–1870 (2014). </li><li> Mi, L., White, A. D., Shao, Q., Setlow, P., Li, Y. & Jiang, S. Chemical insights into dodecylamine spore lethal germination. Chemical Science 5, 3320–3324 (2014). </li><li> White, A. D. & Voth, G. A. Efficient and minimal method to bias molecular simulations with experimental data. Journal of chemical theory and computation 10, 3023–3030 (2014). </li><li> White, A. D., Keefe, A. J., Nowinski, A. K., Shao, Q., Caldwell, K. & Jiang, S. Standardizing and simplifying analysis of peptide library data. Journal of chemical information and modeling 53, 493–499 (2013). </li><li> Brault, N. D., White, A. D., Taylor, A. D., Yu, Q. & Jiang, S. A directly functionalizable surface platform for protein arrays in undiluted human blood plasma. Analytical Chemistry 85, 1447–1453 (2013). </li><li> Keefe, A. J., Caldwell, K. B., Nowinski, A. K., White, A. D., Thakkar, A. & Jiang, S. Screening nonspecific interactions of peptides without background interference. Biomaterials 34, 1871–1877 (2013). </li><li> White, A. D., Keefe, A. J., Ella-Menye, J.-R., Nowinski, A. K., Shao, Q., Pfaendtner, J. & Jiang, S. Free energy of solvated salt bridges: A simulation and experimental study. The Journal of Physical Chemistry B 117, 7254–7259 (2013). </li><li> Nowinski, A. K., Sun, F., White, A. D., Keefe, A. J. & Jiang, S. Sequence, structure, and function of peptide self-assembled monolayers. Journal of the American Chemical Society 134, 6000–6005 (2012). </li><li> White, A. D., Nowinski, A. K., Huang, W., Keefe, A. J., Sun, F. & Jiang, S. Decoding nonspecific interactions from nature. Chemical Science 3, 3488–3494 (2012). </li><li> Shao, Q., He, Y., White, A. D. & Jiang, S. Different effects of zwitterion and ethylene glycol on proteins. The Journal of chemical physics 136, (2012). </li><li> White, A. D., Huang, W. & Jiang, S. Role of nonspecific interactions in molecular chaperones through model-based bioinformatics. Biophysical Journal 103, 2484–2491 (2012). </li><li> White, A. & Jiang, S. Local and bulk hydration of zwitterionic glycine and its analogues through molecular simulations. The Journal of Physical Chemistry B 115, 660–667 (2011). </li><li> Shao, Q., He, Y., White, A. D. & Jiang, S. Difference in hydration between carboxybetaine and sulfobetaine. The Journal of Physical Chemistry B 114, 16625–16631 (2010). </li><li> Yang, W., Zhang, L., Wang, S., White, A. D. & Jiang, S. Functionalizable and ultra stable nanoparticles coated with zwitterionic poly (carboxybetaine) in undiluted blood serum. Biomaterials 30, 5617–5621 (2009). </li> </ol> </div>  </body></html>